plugins/events-node/src/api/DefaultEventsService.ts

```diff
@@ -227,7 +228,14 @@ class PluginEventsService implements EventsService {
           // immediately again
           if (res.status === 202) {
             lock.release();
-            await res.body?.getReader()?.closed;
+            await Promise.race([
+              res.body?.getReader()?.closed,
```

Comment:
If this is hanging we'll likely want to do more than ignore it. Probably best to close the previous request if we hit the timeout.

Reply:
I'm assuming this involves passing an Abort controller signal to the underlying fetch call? If so, I'm not too sure how to go about that here since we're using a `DefaultApiClient` that is auto generated through the openapi spec and I'm unfamiliar with how to extend that interface - mind providing some guidance? 🙏